src/mixins/rippleable.js

```diff
@@ -4,16 +4,23 @@ import Ripple from '../directives/ripple'
 export default {
   directives: { Ripple },
 
+  props: {
+    ripple: {
+      type: [Boolean, Object],
```

Comment:
Why is this an object too?

Reply:
This was copied from VBtn.  I don't know why it was done there, as it's different from Routeable's ripple prop.  I chose to stay consistent with VBtn.  I could go either way.